tests/conftest.py

```diff
@@ -280,16 +280,35 @@ def ir_compiler(ir, *args, **kwargs):
     return ir_compiler
 
 
+def _get_output_formats(experimental_codegen):
+    """
+    Returns a list of output formats supported. If not running the experimental
+    codegen path, the bb and bb_runtime formats are removed.
+    """
+    output_formats = list(compiler.OUTPUT_FORMATS.keys())
+    if experimental_codegen is False:
```

Comment:
i don't really think we need to thread the experimental flag through the test suite yet. let's just always remove `bb` and `bb_runtime` during tests.